lib/sportradar/api/nfl/team.rb

```diff
@@ -21,6 +20,13 @@ def initialize(data)
         @venue = Sportradar::Api::Nfl::Venue.new data["venue"] if data["venue"]
         @hierarchy = Sportradar::Api::Nfl::Hierarchy.new data["hierarchy"] if data["hierarchy"]
 
+        @wins = data["wins"]
+        @losses = data["losses"]
+        @losses = data["losses"]
+        @ties = data["ties"]
+        @win_pct = data["win_pct"].to_f
```

Comment:
Change to `@win_pct = data["win_pct"].to_f if data["win_pct"]` to prevent `nil.to_f => 0.0`